resources/monitoring/values.yaml

```diff
@@ -1223,9 +1290,6 @@ prometheusOperator:
     patch:
       enabled: false
       image:
-        repository: jettech/kube-webhook-certgen # This image is not activated in Kyma. This is kept here to keep parity with upstream charts
-        tag: v1.5.0
```

Comment:
please read the comment... please just keep it commented, the image is not actively used

Reply:
The image was restored